uint8 NONE = 0
uint8 HERO = 1
uint8 ENGINEER = 2
uint8 INFANTRY3 = 3
uint8 INFANTRY4 = 4
uint8 INFANTRY5 = 5
uint8 OUTPOST = 6
uint8 SENTRY = 7
uint8 BASE = 8

uint8 data